packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
 ---
-title: Internal Tools
+title: Access internal tools
 ---
 
 # Internal Tools
 
-This document describes tools available internally to the vets.gov team and configuration required to access them. You'll need access to these tools if you need:
+These instructions cover the configuration of a SOCKS proxy necessary for access to:
 
-* Build logs and details from Jenkins (linked to from GitHub PRs for each project)
+* Static `vets-website` content
+* Remote build logs for pull requests and deployments (Jenkins)
 * System metrics for diagnostic/troubleshooting purposes
 * Exception reports and tracebacks
 
-These internal tools are available on the `vetsgov-internal` domain, which is only accessible while your system is running a SOCKS proxy locally. Your SOCKS proxy will tunnel traffic over a secure channel to vets.gov servers, providing access to:
-
-| Name                 | URL                                              |
-|----------------------|--------------------------------------------------|
-| Jenkins              | http://jenkins.vetsgov-internal                  |
-| Grafana              | http://grafana.vetsgov-internal                  |
-| Sentry               | http://sentry.vetsgov-internal                   |
-| Prometheus (dev)     | http://prometheus-dev.vetsgov-internal:9090/     |
-| Prometheus (staging) | http://prometheus-staging.vetsgov-internal:9090/ |
-| Prometheus (prod)    | http://prometheus-prod.vetsgov-internal:9090/    |
-| Prometheus (utility) | http://prometheus-utility.vetsgov-internal:9090/ |
-
-You do not need to run the SOCKS proxy while you're developing unless you need access to one of the above tools.
-
-## Design Rationale
```

Comment:
Design rationale has been moved to a design decision since it's not relevant to setting up the SOCKS proxy.